src/graphic/animation/spritesheet_animation.cc

```diff
@@ -131,8 +131,8 @@ SpriteSheetAnimation::SpriteSheetMipMapEntry::frame_textures(bool return_playerc
 	std::vector<std::unique_ptr<const Texture>> result;
 	const Rectf rect(Vector2f::zero(), width(), height());
 	if (!return_playercolor_masks || has_playercolor_masks) {
-		const size_t no_of_frames = rows * columns;
-		for (size_t i = 0; i < no_of_frames; ++i) {
+		const int no_of_frames = rows * columns;
```

Comment:
as int is only in the best case as big as size_t this change is not understandable and it does not cure for the potential multiplication overflow.